plugins/internal/tengoutil/secure_script.go

```diff
@@ -0,0 +1,21 @@
+package tengoutil
```

Comment:
Planning to use it for HTTP extractor as well so it will live very temporarily in `plugins/processors` .. should be okay to keep it in current location IMO

Reply:
I see, sure then let's keep it this way, I don't have a really strong opinion on it aside from keeping it the code close where it is being used.